test/extensions/filters/network/redis_proxy/conn_pool_impl_test.cc

```diff
@@ -329,6 +392,24 @@ TEST_F(RedisConnPoolImplTest, HostRemove) {
   testing::Mock::AllowLeak(host2.get());
 }
 
+// This test removes a host from a ConnPool that was never added in the first place. No errors
+// should be encountered.
+
```

Comment:
nit: del line between comment and test, same elsewhere

Reply:
ok